### ISSUE TITLE
Kubernetes ConfigMaps in podman

### ISSUE THREAD

**Issue by dtitov** [containers/podman#7567]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

As per the suggestion from this [comment](https://github.com/containers/podman/issues/3264#issuecomment-689449505), I'm creating a feature request here for implementing something similar to [Kubernetes ConfigMaps](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) in podman. 

Basically, there's a need for a convenient way of propagating environment variables to the `podman play kube` command.

**Comment by rhatdan** [comment]
@haircommander WDYT?

**Comment by haircommander** [comment]
makes sense to me

**Comment by EduardoVega** [comment]
@rhatdan,  @haircommander I would like to help on this one.

**Comment by haircommander** [comment]
thank you @EduardoVega !

**Comment by rhatdan** [comment]
Sounds great @EduardoVega
